src/components/TopBar/TopBar.vue

```diff
@@ -438,6 +441,10 @@ export default {
 	}
 }
 
+.description__popover {
+	padding: var(--default-grid-baseline);
```

Comment:
Check with Marco, I remember that we agreed on 50em for text width
```suggestion
	padding: var(--default-grid-baseline);
	width: fit-content;
	max-width: 50em;
```